docs/source/index.rst

```diff
@@ -17,11 +17,37 @@ through Python or directly on the command line.
    status = k.status(app_id)
    logs = k.logs(app_id)
 
+Motivation
+----------
+
+``Knit`` is built to support long-running non JVM based applications.  For example, Knit can
+deploy Python based distributed applications such as `IPython Parallel`_
+
+*  **Convenience:** General purpose YARN applications with easy to use CLI and Python interface
+*  **PyData Support** Bring PyData stack into the Hadoop/YARN ecosystem
+*  **Deployable Runtimes:** Knit can build and ship self contained runtime environments.  This means
+   we can resolve library dependencies without significant IT infrastructure
+   and management
```

Comment:
Stick to all adjective phrases or all full sentences with verbs

Convenience is a phrase while PyData Support is verby.  Personally I like starting with verbs.

*  Launch YARN applications with an easy to use CLI and Python interface
*  Bring the PyData stack into the Hadoop/YARN ecosystem
*  Build and Ship self contained runtime environments.  

Although this may sound too salesy